### PR TITLE
fix: redirect user after `loggedIn()` and called `/login/magic-link` page

### PR DESCRIPTION
**Pull request by datamweb**
Hello good guys
Ref: #215 
It seems that changes should have been made here as well.

### CODE REVIEW

File: tests/Controllers/MagicLinkTest.php [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+
+        helper('auth');
+
+        // Add auth routes
+        $routes = service('routes');
+        auth()->routes($routes);
+        Services::injectMock('routes', $routes);
+    }
+
+    public function testAfterLoggedInNotAllowDesplayMagicLink()
```
Comment (kenjis) [inline tests/Controllers/MagicLinkTest.php:34]: I found a typo.

```suggestion
    public function testAfterLoggedInNotAllowDisplayMagicLink()
```

Also `LoginTest::testAfterLoggedInNotDesplayLoginPage`.

Comment (datamweb) [reply tests/Controllers/MagicLinkTest.php]: Thanks,
Can I apply here or should I apply in PR separate?

Comment (kenjis) [reply tests/Controllers/MagicLinkTest.php]: > One thing at a time: A pull request should only contain one change. That does not mean only one commit, but one change - however many commits it took. The reason for this is that if you change X and Y, but send a pull request for both at the same time, we might really want X but disagree with Y, meaning we cannot merge the request. Using the Git-Flow branching model you can create new branches for both of these features and send two requests.
https://github.com/codeigniter4/CodeIgniter4/blob/develop/contributing/pull_request.md#branching

**Review by kenjis — APPROVED** [review]

**Review by MGatner — APPROVED** [review]
Not sure what the Rector errors are - got a similar error dump on a different repo also. @samsonasik any ideas?

**Comment by samsonasik** [comment]
@MGatner possibly related with node stmts refresh on multi rules, should be fixed at rector/rector:dev-main ref https://github.com/rectorphp/rector-src/pull/2453

**Comment by MGatner** [comment]
That looks likely! Glad you got a fix in, we can wait it out.

**Pull request merged**